addon/components/bs-modal.js

```diff
@@ -193,6 +193,20 @@ export default class Modal extends Component {
   @arg
   scrollable = false;
 
+    /**
+   *  Allows adding fullscreen mode for modals. It will
+   *  apply the `modal-fullscreen' class when using 'true' and
+   *  `modal-fullscreen-[x]-down` class when using BS breakpoints
+   *   ([x] = 'sm', 'md', 'lg', 'xl', 'xxl').
+   *  
+   * @property fullscreen
+   * @type {string}
+   * @default 'true'
+   * @public
+   */
+  @arg
+  fullscreen = 'true';
```

Comment:
This doesn't seem to be used at all as you are not accessing `{{this.fullscreen}}` in the template but only `{{@fullscreen}}`. I think it could be removed safely.

```suggestion
```

Reply:
Fixed